File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/ocm/OxSession.java
package com.gigigo.orchextra.core.domain.entities.ocm;

public class OxSession {
  private String key;
  private String secret;
  private String token;

  public void setCredentials(String key, String secret) {
    this.key = key;
    this.secret = secret;
  }

  public String getKey() {

    return key;
  }

  public String getSecret() {
    return secret;
  }

  public void setToken(String token) {
    this.token = token;
  }

  public String getAccessToken() {
    return "Bearer " + token;
  }
}

File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/ocm/OxSession.kt
package com.gigigo.orchextra.core.domain.entities.ocm

class OxSession {
  var key: String? = null
    private set

  var secret: String? = null
    private set

  var token: String? = null

  val accessToken: String
    get() = "Bearer " + token!!

  fun setCredentials(key: String, secret: String) {
    this.key = key
    this.secret = secret
  }
}